pkg/controllers/node/termination/terminator/eviction.go

```diff
@@ -123,7 +128,9 @@ func (q *Queue) Has(pod *v1.Pod) bool {
 	return q.set.Has(NewQueueKey(pod))
 }
 
-func (q *Queue) Reconcile(ctx context.Context, _ reconcile.Request) (reconcile.Result, error) {
+func (q *Queue) Reconcile(ctx context.Context) (reconcile.Result, error) {
+	ctx = injection.WithControllerName(ctx, "eviction-queue")
```

Comment:
Can we align on if we're doing kabob case or doing `.` across all controllers?

Reply:
I'd agree we should align on this and ideally have this match the file directory structure; however, this isn't a change that's being introduced in this PR (it's only maintaining the current functionality), so I'd like to decouple things so that we make that change separately

Reply:
Fine with this, just wondering what the reasonable difference is here? Is it that logging would be slightly different for those who are looking at the eviction queue? Seems like maybe it's just this file, which we could just change it to be . instead of -.

Reply:
> Seems like maybe it's just this file, which we could just change it to be . instead of -.

We can decouple and follow-up on this in a follow-up PR IMO